### PR TITLE
Improvement/sdk upgrade

### PR DESCRIPTION
**Pull request by rfreitas**
# Problem

App was using an old version of the sdk which did not upload the captures to the onfido api.

# Solution

1.  Upgraded to latest sdk
2. Generate the new jwt version (create a new applicant on every refresh)

# Technical details

1. Removed web dev server
2. Made server babel code
3. Brought in simple rest client

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,19 +7,13 @@ window.onload = function() {
   request.onload = function() {
     if (request.status >= 200 && request.status < 400) {
       var data = JSON.parse(request.responseText)
-
+      console.log("JWT response", data)
```
Comment (hjheath) [inline src/index.js]: Remove?

File: src/index.js [hunk 2]
```diff
@@ -7,19 +7,13 @@ window.onload = function() {
...
       window.onfidoOut = Onfido.init({
         useModal: false,
-        token: data.jwt,
+        token: data.token,
         onReady: function() {
```
Comment (hjheath) [inline src/index.js]: remove `onReady` too?

File: app/server.js [hunk 3]
```diff
@@ -1,85 +1,76 @@
...
+  createApplicant( ({body:applicant}) => {
+    onfidoRequest('post','sdk_token',
+      {referrer:"*://*/*", applicant_id:applicant.id},
+      callback
+    )
+  })
+}
 
-var isOriginHostnameSameAsServer = function(req){
+const isOriginHostnameSameAsServer = function(req){
```
Comment (hjheath) [inline app/server.js]: I don't think this function is used anywhere - remove

**Comment by hjheath** [comment]
Looks good, I think you can remove even more code (per comments).

FYI we have a node API client library, although I'm not sure it has the `sdk_token` endpoint. https://github.com/onfido/api-javascript-client

File: package.json [hunk 4]
```diff
@@ -45,19 +44,18 @@
...
     "url-loader": "0.5.8",
-    "file-loader": "0.10.1",
-    "webpack": "^1.12.14",
-    "webpack-dev-server": "^1.14.1"
+    "webpack": "^1.12.14"
   },
   "dependencies": {
     "css-loader": "^0.23.1",
     "express": "^4.14.0",
+    "onfido-sdk-ui": "^1.1.0",
```
Comment (dawiddylowicz) [inline package.json]: Make the version fixed so that we don't break sample app with every onfido-sdk-ui release update

**Comment by rfreitas** [comment]
@hjheath @dawiddylowicz feedback pushed

**Pull request merged**